test/EmailAddressTest.php

```diff
@@ -741,8 +741,8 @@ public function testUseMxCheckBasicValid()
         ];
 
         if (extension_loaded('intl')) {
-            $emailAddresses[] = 'иван@письмо.рф';
```

Comment:
Yep. But this is part of another PR to make this validator RFC 6532 compatible (coming soon).